iree_tests/configs/models_gpu_rocm_gfx90a.json

```diff
@@ -2,25 +2,23 @@
     "config_name": "gpu_rocm",
     "iree_compile_flags" : [
       "--iree-hal-target-backends=rocm",
-      "--iree-rocm-target-chip=gfx90a",
-      "--iree-opt-const-eval=false",
-      "--iree-codegen-transform-dialect-library=${IREE_TEST_PATH_EXTENSION}/attention_and_matmul_spec.mlir"
+      "--iree-rocm-target-chip=gfx90a"
     ],
     "iree_run_module_flags": [
       "--device=hip"
     ],
-    "skip_compile_tests": [
-      "pytorch/models/sdxl-scheduled-unet-3-tank",
-      "pytorch/models/sdxl-vae-decode-tank",
-      "pytorch/models/sdxl-prompt-encoder-tank"
-    ],
+    "skip_compile_tests": [],
     "skip_run_tests": [],
     "expected_compile_failures": [
-      "pytorch/models/opt-125M", // TODO(#17344): need to regenerate .mlirbc
-      "pytorch/models/resnet50",
+      "pytorch/models/direct/opt-125M", // TODO(#17344): need to regenerate .mlirbc
+      "pytorch/models/direct/resnet50",
       // error: 'builtin.module' op failed to run transform dialect passes
       // (might need to drop the iree-codegen-transform-dialect-library flag)
```

Comment:
Can drop this comment since the `attention_and_matmul_spec` is removed.
I also saw `error: a handle passed as operand #0 and consumed by this operation points to a payload entity more than once` and updated the comment in my draft PR: https://github.com/nod-ai/SHARK-TestSuite/pull/277

How about we just leave the reason comments off? Also on my PR there, if we run with `-rA` (instead of `-rpfE`, see https://docs.pytest.org/en/stable/how-to/output.html), we'll get full error output for even XFAIL'd tests in CI logs, so having an possibly outdated comment in the source isn't as useful.